odilia/src/commands.rs

```diff
@@ -7,43 +7,3 @@ type Request = Command;
 type Response = ();
 type Error = OdiliaError;
 
-impl<F, Fut, S, E> Handler<(Request,), S, E> for F
-where
-	F: FnOnce(E) -> Fut + Clone + Send,
-	Fut: Future<Output = Result<Response, Error>> + Send + 'static,
-	S: Clone,
-{
-	type Response = Response;
-	type Future = Fut;
-	fn call(self, req: E, _state: S) -> Self::Future {
-		self(req)
-	}
-}
-
-impl<F, Fut, S, E, T1> Handler<(Request, T1), S, E> for F
-where
-	F: FnOnce(E, T1) -> Fut + Clone + Send,
-	Fut: Future<Output = Result<Response, Error>> + Send + 'static,
-	S: Clone,
```

Comment:
just clone? any type which implements clone? what is this part of the code for anyway?

Reply:
Basically we assume that `S` is state, and that `S` is an `Arc<T>`; if the users of these traits (ourselves) do not realize that we need to wrap `ScreenReaderState` in an `Arc` so that cloning isn't 99% of all our data, that's on us.